user-guide/docs/usecases/padgett/usecase_JN_viz.md

```diff
@@ -12,7 +12,20 @@ Jupyter notebook for visualization of spatially distributed data in risk and res
 _Keywords: visualization; risk and resilience; infrastructure systems; static, interactive, and animated maps and figures, effective communication_
 
 ### Resources
- 
+
+<script type="text/x-mathjax-config">
+MathJax.Hub.Config({  
+jax: ["input/TeX", "output/HTML-CSS"],
+extensions: ["tex2jax.js"],
+"HTML-CSS": { preferredFont: "TeX", availableFonts: ["STIX","TeX"] },
+tex2jax: { inlineMath: [ ["$", "$"], ["\\(","\\)"] ], displayMath: [ ["$$","$$"], ["\\[", "\\]"] ], processEscapes: true, ignoreClass: "tex2jax_ignore|dno" }, 
+TeX: { noUndefined: { attributes: { mathcolor: "red", mathbackground: "#FFEEEE", mathsize: "90%" } } },
+messageStyle: "none"  
+});
+
+</script> 
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js"></script>
+
```

Comment:
As of #109, this should now be done via `{% include-markdown '../../mathjax.md' %}`.